src/deepsparse/transformers/pipelines/embedding_extraction.py

```diff
@@ -100,10 +100,7 @@ def to_list(cls) -> List[str]:
 @Pipeline.register(
     task="transformers_embedding_extraction",
     task_aliases=[],
-    default_model_path=(
-        "zoo:nlp/masked_language_modeling/bert-base/pytorch/huggingface/"
-        "wikipedia_bookcorpus/pruned80_quant-none-vnni"
-    ),
+    default_model_path=("zoo:bert-large-wikipedia_bookcorpus-pruned90"),
```

Comment:
But that's a different model from the initial one?

Reply:
Original models dont exist anymore. 
Replaced by models that share the same task